src/main/java/org/wildfly/security/auth/server/SecurityDomain.java

```diff
@@ -81,6 +83,7 @@
         this.principalDecoder = builder.principalDecoder;
         this.securityIdentityTransformer = builder.securityIdentityTransformer;
         this.trustedSecurityDomain = builder.trustedSecurityDomain;
+        this.scheduledExecutorService = builder.scheduledExecutorService;
```

Comment:
Fixed. I'll close my [elytron-subsystem PR](https://github.com/wildfly-security/elytron-subsystem/pull/197) since that's no longer needed with this change.